language/diem-framework/tests/DiemCurrenciesTests.move

```diff
@@ -1,15 +1,18 @@
 // Check that the add_all_currencies flag does the expected thing
-
-
 //! new-transaction
 //! sender: blessed
-script {
+#[test_only]
+module DiemFramework::DiemCurrenciesTests {
```

Comment:
Could you change the module name from `DiemCurrenciesTests` to `DiemDesignatedDealerTests`? Currency tests are just a sub category under the umbrella.